### PR TITLE
BENCH-72: Compile and build Answer King app using GitHub Actions 

### PR DESCRIPTION
**Pull request by mbaltrimas**
had to update the line endings of all files because of linter failing

### CODE REVIEW

File: src/dtos/OrderItemDto.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export interface OrderItemDto {
+  id: number;
+  name: string;
+  price: number;
```
Comment (harrystead2) [inline src/dtos/OrderItemDto.ts]: We don't want these files added do we?

Comment (harrystead2) [reply src/dtos/OrderItemDto.ts]: And why has it rebuilt all the files?

Comment (mbaltrimas) [reply src/dtos/OrderItemDto.ts]: Oh oops didn't even notice the files being added back - removed them. The line endings of all files were changed from LF to CRLF which is why they show up as affected

Comment (harrystead2) [reply src/dtos/OrderItemDto.ts]: Ah yes, I see! Ok will approve

**Review by harrystead2 — APPROVED** [review]

**Review by hazel-davis — APPROVED** [review]

**Review by bethcryer — APPROVED** [review]

**Pull request merged**